### PR TITLE
OwnAdx: Add cookie sync

### PR DESCRIPTION
**Pull request by ownAdx-prebid**
We are going to compatible for Cookie sync

### CODE REVIEW

File: static/bidder-info/ownadx.yaml [hunk 1]
```diff
@@ -10,3 +10,9 @@ capabilities:
     mediaTypes:
       - banner
       - video
+
+userSync:
+  redirect:
+    url: "https://sync.spoutroserve.com/user-sync?t=image&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&s3={{.RedirectURL}}"
+    userMacro: "{USER_ID}"
```
Comment (onkarvhanumante) [inline static/bidder-info/ownadx.yaml:17]: remove double quotes from userMacro value

```
 userMacro: {USER_ID}
```

Comment (ownAdx-prebid) [reply static/bidder-info/ownadx.yaml]: Double quotes removed.

**Comment by ownAdx-prebid** [comment]
@onkarvhanumante 
As per request we are make changes . Can i know if required more changes

**Comment by gargcreation1992** [comment]
@ownAdx-prebid Build pipeline is failing. Unable to parse ownadx.yaml file. Please check and fix. you can run prebid in your local using - `go build . && ./prebid-server --stderrthreshold=WARNING -v=2` and see if it compiles.

**Comment by ownAdx-prebid** [comment]
We compile project on local, and find...
1) As per our previous discussion we removed double quotes on User Macro value.
But because of that we get following error,
" F0618 11:23:29.323207   64890 main.go:37] Unable to load bidder configurations: error parsing config for bidder ownadx.yaml: 
  yaml: unmarshal errors: line 17: cannot unmarshal !!map into string "

2) So we again add the double quotes on user macro value please verify.

**Review by gargcreation1992 — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**